be/src/exec/vectorized/hdfs_scan_node.cpp

```diff
@@ -577,7 +577,9 @@ Status HdfsScanNode::close(RuntimeState* state) {
 
 Status HdfsScanNode::set_scan_ranges(const std::vector<TScanRangeParams>& scan_ranges) {
     for (const auto& scan_range : scan_ranges) {
-        _scan_ranges.emplace_back(scan_range.scan_range.hdfs_scan_range);
+        auto& range = scan_range.scan_range.hdfs_scan_range;
+        if (range.file_length == 0) continue;
```

Comment:
better to give some comment and why not eliminate these at Frontend.

Reply:
yes, better to remove at frontend. And for safety/backward-compatibility, we can also keep this line.